hudi-cli/hudi-cli.sh

```diff
@@ -20,15 +20,12 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 HOODIE_JAR=`ls $DIR/target/hudi-cli-*.jar | grep -v source | grep -v javadoc`
-if [ -z "$HADOOP_CONF_DIR" ]; then
-  echo "setting hadoop conf dir"
-  HADOOP_CONF_DIR="/etc/hadoop/conf"
-fi
-if [ -z "$SPARK_CONF_DIR" ]; then
-  echo "setting spark conf dir"
-  SPARK_CONF_DIR="/etc/spark/conf"
-fi
+
+. "${DIR}"/conf/hudi-env.sh
+
 if [ -z "$CLIENT_JAR" ]; then
-  echo "client jar location not set"
+  echo "Client jar location not set, please set it in conf/hudi-env.sh"
+  exit
 fi
```

Comment:
hi, may add code snippet is better, WDYT ?
```
if [ -z "$HADOOP_CONF_DIR" ]; then
  echo "hadoop conf dir not set"
fi
if [ -z "$SPARK_CONF_DIR" ]; then
  echo "spark conf dir not set"
fi
```

Reply:
> hi, may add code snippet is better, WDYT ?
> 
> ```
> if [ -z "$HADOOP_CONF_DIR" ]; then
>   echo "hadoop conf dir not set"
> fi
> if [ -z "$SPARK_CONF_DIR" ]; then
>   echo "spark conf dir not set"
> fi
> ```

All of they are export with a default path when run hudi-env.sh, so they almost impossible to be null.

Reply:
> hi, may add code snippet is better, WDYT ?
> 
> ```
> if [ -z "$HADOOP_CONF_DIR" ]; then
>   echo "hadoop conf dir not set"
> fi
> if [ -z "$SPARK_CONF_DIR" ]; then
>   echo "spark conf dir not set"
> fi
> ```

All of they are export with a default path when run hudi-env.sh, so they almost impossible to be null.